### PR TITLE
Add Blake2b-160

### LINKED ISSUE #110: Implement BLAKE2b-160
Currently, the only supported version is BLAKE2b-256. This should be extended to also support the 160-bit version of BLAKE2b:
- Add a new wrapper for VarBlake2b with 160-bit output length, matching `Blake2b256`.
- Add test vectors, at least using `b2sum --length 160`
- Add BLAKE2b-160 to the _List of Algorithms_

### PR DESCRIPTION
**Pull request by vuongDang**
- Additional wrapper for VarBlake2b
  - Additional tests for 160 bits length
  - Update list of supported algorithms

# Description of change
Support for Blake2b for 160 bits long hash
- Write wrapper for VarBlake2b
- Generate new tests based on the previous tests of blake2b-256
- New test vectors created with b2sum using the inputs of blake2b-256 test vector
- Update list of supported algorithms 

## Links to any relevant issues

Fixes issue #110

## Type of change

Choose a type of change, and delete any options that are not relevant.

- [x] Enhancement (a non-breaking change which adds functionality)
- [x] Documentation 

## How the change has been tested

Added new tests which are an adaptation of the previous blake2b-256
Passed the `cargo test --features blake2b`

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ To be included in this list an implementation must:
 | ciphers     | AES-KW        | [`aes-kw`](/src/ciphers/aes_kw.rs)        | [spec][AES-GCM-SPEC]       | `aes-crate`          | [nist][AES-GCM-TEST]     | ★★★☆☆ |
 | ciphers     | XCHACHA20-POLY1305 | [`chacha`](/src/ciphers/chacha.rs)  | [rfc][XCHACHA-RFC]         | `chacha20poly1305` | [official][XCHACHA-TEST] | ★★★★★ |
 | hashes       | BLAKE2b-256        | [`blake2b`](/src/hashes/blake2b.rs) | [rfc][BLAKE2B-RFC]         | `blake2`           | [official][BLAKE2B-TEST] | ★★★★☆ |
+| hashes       | BLAKE2b-160        | [`blake2b`](/src/hashes/blake2b.rs) | [rfc][BLAKE2B-RFC]         | `blake2`           | [official][BLAKE2B-TEST] | ★★★★☆ |
```
Comment (Wollac) [inline README.md:28]: Let's put this before the BLAKE2b-256 so that everything is sorted

**Review by Wollac — CHANGES_REQUESTED** [review]
Missing a `.changes` file. Please add one; consult that directory for more information.

**Review by Wollac — APPROVED** [review]
Thanks. Looks good to me

**Review by semenov-vladyslav — APPROVED** [review]
Looks good

**Pull request merged**